Packages/com.chisel.editor/Chisel/Editor/Editor/Scene/CSGSceneBottomGUI.cs

```diff
@@ -13,670 +13,98 @@ namespace Chisel.Editors
 	// TODO: add tooltips
 	public static class CSGSceneBottomGUI
 	{
-		const float kBottomBarHeight    = 18;
-		const float kButtonSpace        = 6;
-		const float kButtonWidth        = 18;
-		const float kDropdownWidth      = kButtonWidth + 8;
+		const float kBottomBarHeight    = 20;
 		const float kTopBarHeight       = 17;
 		const float kFloatFieldWidth    = 60;
 
 		static readonly int BottomBarGUIHash = typeof(CSGSceneBottomGUI).Name.GetHashCode();
-		static GUIStyle miniLabel;
-		static GUIStyle lockXButtonStyle;
-		static GUIStyle lockYButtonStyle;
-		static GUIStyle lockZButtonStyle;
-		static Color[]  lockButtonToggleColor;
-		static bool     prevSkin = false;
-		static GUIContent rebuildContent;
-		static GUIContent viewContent;
-
-		static readonly GUIContent lockContent              = new GUIContent("Lock");   
-		static readonly GUIContent lockXAxisContent         = new GUIContent("X");
-		static readonly GUIContent lockYAxisContent         = new GUIContent("Y");
-		static readonly GUIContent lockZAxisContent         = new GUIContent("Z");
-
-		static readonly GUIContent boundsSnappingContent	= new GUIContent("Snap-Bounds");
-		static readonly GUIContent pivotSnappingContent		= new GUIContent("Snap-Pivot");
```

Comment:
being able to toggle bounds and pivot snapping is very useful to have on the bottom bar imho